### PR TITLE
{vis}[GCCcore/8.3.0,foss/2019b,intel/2019b] Grace v5.1.25, motif v2.3.8

### PR DESCRIPTION
**Pull request by migueldiascosta**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by migueldiascosta** [comment]
Test report by @migueldiascosta
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in this PR)
generoso - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2660 v4 @ 2.00GHz, Python 3.6.8
See https://gist.github.com/5dcb7ccb0d774e3109583e65ddb1ec0c for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in this PR)
node3131.skitty.os - Linux centos linux 7.7.1908, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 3.6.8
See https://gist.github.com/f43307efeca9faa03eabf310f7e48e04 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in this PR)
node2557.golett.os - Linux centos linux 7.7.1908, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/705b666ddef1c44a191e04bb7fd38352 for a full test report.

File: easybuild/easyconfigs/g/Grace/Grace-5.1.25-intel-2019b-5build1.eb [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+checksums = [
+    '751ab9917ed0f6232073c193aba74046037e185d73b77bab0f5af3e3ff1da2ac',  # grace-5.1.25.tar.gz
+    '485fe80ef992e2068f5aa4bc94c0968b5bbef5491ff7c60c56152b61dcacbd46',  # Grace-5.1.25-5build1.patch
+]
+
+dependencies = [
+    ('motif', '2.3.8'),
+    ('zlib', '1.2.11'),
+    ('FFTW', '3.3.8'),
+    ('netCDF', '4.7.1'),
```
Comment (boegel) [inline easybuild/easyconfigs/g/Grace/Grace-5.1.25-intel-2019b-5build1.eb]: @migueldiascosta Why is the list of dependencies different here compared to the `foss` toolchain?

Wouldn't it be better to align them?

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in this PR)
node3139.skitty.os - Linux centos linux 7.7.1908, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 3.6.8
See https://gist.github.com/bd41d1a9ba1951104135e364de3ee6c1 for a full test report.

**Comment by boegel** [comment]
@migueldiascosta Looks like this needs a sync with `develop` after #9954 was merged... The checksum for the `rgeos` easyconfig did get added in #9954, but it's not in your branch yet which is making the tests fail. :-/

**Comment by boegel** [comment]
Going in, thanks @migueldiascosta!

**Pull request merged**